app/graphql/resolvers/survey_resolver.rb

```diff
@@ -2,12 +2,12 @@
 
 module Resolvers
   class SurveyResolver < BaseResolver
-    description 'Survey list'
+    description 'Survey details'
+    argument :id, ID, required: true
+    type Types::SurveyType, null: true
 
-    type [Types::SurveyType], null: false
-
-    def resolve
-      Survey.all
+    def resolve(id:)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/634d923a312cee5a5351ccd312a92b5bc125132a/app/graphql/resolvers/survey_resolver.rb#L9"></span>Doesn’t depend on instance state (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>